WordPressVIPMinimum/Sniffs/Functions/CreateFunctionSniff.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\Functions;
+
+use PHP_CodeSniffer_File as File;
+use PHP_CodeSniffer_Tokens as Tokens;
+
+/**
+ * This function generates an error when
+ * create_function() is found in code.
+ * create_function() is deprecated in PHP 7.2.0.
+ *
+ * An example:
+ *
+ * <code>
+ *   create_function( 'foo', 'return "bar";' );
+ * </code>
+ *
+ * See here: http://php.net/manual/en/function.create-function.php
+ */
+class CreateFunctionSniff implements \PHP_CodeSniffer_Sniff {
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * We want everything function-related.
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return Tokens::$functionNameTokens;
+
+	}
+
+	/**
+	 * Get next T_STRING token.
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file where the token was found.
+	 * @param int                         $stackPtr  The position in the stack where
+	 *                                               the token was found.
+	 *
+	 * @return int
+	 */
+	private function getNextTString( File $phpcsFile, $stackPtr ) {
+		return $phpcsFile->findNext(
+			T_STRING,
+			( $stackPtr ),
+			null,
+			false,
+			null,
+			true
+		);
+	}
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file where the token was found.
+	 * @param int                         $stackPtr  The position in the stack where
+	 *                                               the token was found.
+	 *
+	 * @return void
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+		$this->_tokens    = $phpcsFile->getTokens();
```

Comment:
I'm not really sure we need to expose `_tokens`, `_phpcsFile` and `_stackPrt` as class properties, as they really are only used in terms of the `process` method (eg.: the `getNextTString` method expects PHP_CodeSniffer_File as it's param, despite it's present in the class property).